changelog/unreleased/issue-17009.toml

```diff
@@ -0,0 +1,5 @@
+type = "fixed"
+message = "Strict file permissions for key material in datanode."
```

Comment:
As a rule of thumb, I wouldn't create changlogs for things that are still not released yet.

Reply:
Good point, removed, thx!